db/config.cc

```diff
@@ -515,6 +515,8 @@ db::config::config(std::shared_ptr<db::extensions> exts)
         "\n"
         "Related information: Failure detection and recovery")
     , failure_detector_timeout_in_ms(this, "failure_detector_timeout_in_ms", liveness::LiveUpdate, value_status::Used, 20 * 1000, "Maximum time between two successful echo message before gossip mark a node down in milliseconds.\n")
+    , direct_failure_detector_ping_timeout_in_ms(this, "direct_failure_detector_ping_timeout_in_ms", liveness::LiveUpdate, value_status::Used, 600, "Duration after which the direct failure detector aborts a ping message, so the next ping can start.\n"
```

Comment:
Why increase it *and* make it tunable? 
Shouldn't making it tunable be sufficient?

Anyway, for the purpose of local tests we should actually keep the low timeout, so please change the default in scylla.yaml used in the tests - this will make the tests a bit more quick.

Reply:
I increased the default so that multi-DC setups don't have the "periodic alive/dead" problem by default.

I believe this won't affect tests in any way.